frontend/src/bundles/common/components/password-input/styles.module.scss

```diff
@@ -13,3 +13,12 @@
     cursor: pointer;
     transform: translateY(-50%);
 }
+
+.password_form > span {
+    bottom: -35px;
+    margin-bottom: 5px;
+}
```

Comment:
space applies only to a password's span, however, the other fields are "attached" to the input

Reply:
fixed